arangod/Replication/Syncer.cpp

```diff
@@ -165,27 +166,29 @@ arangodb::Result applyCollectionDumpMarkerInternal(
 
         // need to replace the one we have
         useReplace = true;
-        opRes.result.reset(TRI_ERROR_NO_ERROR, keySlice.copyString());
+        potentiallyConflictingKey = keySlice.stringView();
```

Comment:
No need for a temporary std::string_view here that is instantly converted into an std::string anyway.
```suggestion
        potentiallyConflictingKey = keySlice.copyString();
```

Reply:
done